### PR TITLE
get explore url for choropleth charts

### PR DESCRIPTION
**Pull request by chejennifer**
add explore url for choropleth charts which will redirect to timeline tool for the stat var for the current place

Changes:
- updated api/chart/choroplethdata/<dcid> to return the explore url for each stat var
- updated chart.tsx to get explore url for choropleth charts

### CODE REVIEW

File: server/routes/api/chart.py [hunk 1]
```diff
@@ -544,7 +544,9 @@ def choropleth_data(dcid):
             sv_denom_result = {
                 'date': sv_date,
                 'data': geo_data,
-                'numDataPoints': num_data_points
+                'numDataPoints': num_data_points,
+                # TODO (chejennifer): exploreUrl should link to choropleth tool once the tool is ready
+                'exploreUrl': build_url(dcid, [sv])
```
Comment (beets) [inline server/routes/api/chart.py:549]: don't forget to set is_scaled=true if there is a denominator

**Review by beets — CHANGES_REQUESTED** [review]

File: server/routes/api/chart.py (hunk 1 above)
Comment (chejennifer) [reply server/routes/api/chart.py:549]: it doesn't look like there's anywhere to set that?

Comment (beets) [reply server/routes/api/chart.py:549]: oh sorry! i thought i had merged #488

File: server/routes/api/chart.py [hunk 2]
```diff
@@ -541,12 +542,17 @@ def choropleth_data(dcid):
                 else:
                     geo_data[geo] = sv_data[geo][sv_date]
                     num_data_points += 1
+            exploreUrl = None
```
Comment (beets) [inline server/routes/api/chart.py:545]: could you just call:

exploreUrl = landing_page_api.build_url([dcid], [sv], bool(denom_data))

**Review by beets — CHANGES_REQUESTED** [review]

**Review by beets — APPROVED** [review]

**Pull request merged**